test/Microsoft.SqlTools.ServiceLayer.Test/LanguageServer/SqlCompletionItemTests.cs

```diff
@@ -13,6 +13,304 @@ namespace Microsoft.SqlTools.ServiceLayer.Test.LanguageServer
 {
     public class SqlCompletionItemTests
     {
+        private static readonly string[] ReservedWords = new string[]
+        {
+            "all",
```

Comment:
I thought we already had this list in the language server code.  Perhaps it makes more sense to have the test access the list from there rather than duplicate it into the test code.